### DISCUSSION TITLE
Support for OnPrem

### DISCUSSION THREAD

**Discussion by reuvygroovy** [pnp/powershell#666] [General] [answered]
I see here you guys write: 
_This version of the cmdlets is cross-platform (e.g. it works on Windows, MacOS and Linux) however will only support SharePoint Online._

Will the old module continue to be available/downloadable for use with OnPrem environments? Is there a chance that at some point they may not work any longer?

**Comment by LeonArmston** [comment, marked as answer]
Hi 

The old [SharePointPnPPowerShell](https://github.com/pnp/PnP-PowerShell/releases) module is going to be continually available but is no longer maintained and is read-only so there will be no further updates.

It should continue to work with on-prem as doubt anything drastic behind the scenes with updates on-prem will happen just no further functionality will be added.

Thanks
